aiida-registry-app/src/Components/Details.jsx

```diff
@@ -93,6 +93,9 @@ function Details({pluginKey}) {
           ))}
             </>
           )}
+          <Alert severity="info">
```

Comment:
is this alert needed or is this obvious?

Reply:
I think this is needed, otherwise, the text is not so apparent in the info box.